content/docs/1.6.4/deploy/important-notes/index.md

```diff
@@ -224,14 +224,14 @@ Longhorn v{{< current-version >}} does not allow the following:
 However, Longhorn still allows the listed actions when cloning or restoring volumes created with earlier Longhorn
 versions.
 
-### Backup Related Custom Resources Might Disappear
+### Backup Data On The Remote Backup Server Might Be Deleted
 
-Longhorn will attempt to clean up the backup-related custom resources in the following scenarios:
+Longhorn will attempt to delete backup data on the remote backup server and clean up the backup-related custom resources (BackupVolume, BackupBackingImage, SystemBackup, and Backup) of the cluster in the following scenarios:
 
 - An empty response from the NFS server due to a brief server downtime.
-- A race condition between related Longhorn backup controllers.
+- There is a race condition that could potentially delete the remote backup volume and its corresponding backups when resetting the backup target within a short period.
 
-The backup information will be resynchronized during the next polling interval.  
+From Longhorn v{{< current-version >}}, the remote backup data will not be deleted with the synchronization and backup information will be resynchronized during the next polling interval.  
```

Comment:
```suggestion
Starting with v{{< current-version >}}, Longhorn does not delete the remote backup data during synchronization. Instead, the backup data is resynchronized during the next polling period.
```

Reply:
The draft is unclear so I broke up the sentence where it makes sense. Let me know if I misunderstood.

Reply:
Modified, and please review it again. Thanks.